frontend/src/style/partials/header.less

```diff
@@ -14,6 +14,10 @@
   .last {
     margin-right: 30px;
   }
+
+  .cursor {
```

Comment:
```suggestion
  .cursor-pointer {
```

Reply:
oops, my bad. This is indeed the correct way, I don't know how but I forgot... Thanks brother.